2022-05-metafacture-workshop/index.html

```diff
@@ -0,0 +1,944 @@
+			</section>
+
+			<section>
+				<h2>TODO: Playground-Export/CLI/Import &amp; Übung</h2>
+			</section>
+
+			<section>
+				<h1>Rückschau</h1>
+				<p><span class="fragment">Workflows mit Flux</span></p>
+				<p><span class="fragment">Transformationen mit Fix</span></p>
+				<p><span class="fragment">Playground, Export, Import</span></p>
+			</section>
+
+			<section>
+				<h1>Daten analysieren</h1>
+			</section>
+
+			<section>
+				<h1>stream-to-triples</h1>
+				<img src="img/stream-to-triples.png"/>
+				<p><small>Source: Christoph Böhme, <a href="http://swib.org/swib13/slides/boehme_swib13_131.pdf">http://swib.org/swib13/slides/boehme_swib13_131.pdf</a></small></p>
+			</section>
+
+			<section>
+				<h1>count-triples</h1>
+				<img src="img/count-triples.png"/>
+				<p><small>Source: Christoph Böhme, <a href="http://swib.org/swib13/slides/boehme_swib13_131.pdf">http://swib.org/swib13/slides/boehme_swib13_131.pdf</a></small></p>
+			</section>
+
+			<section>
+				<h1>template</h1>
+				<img src="img/template.png"/>
+				<p><small>Source: Christoph Böhme, <a href="http://swib.org/swib13/slides/boehme_swib13_131.pdf">http://swib.org/swib13/slides/boehme_swib13_131.pdf</a></small></p>
+			</section>
+
+			<section>
+				<h2>Beispiel: Feldwerte zählen</h2>
+				<p>Anzahl unterschiedlicher Werte im Edition-Feld (032@.a)</p>
+				<span class="fragment">
+				<pre><code class="bash" data-trim>
+$ gedit count.fix &amp;
+				</code></pre>
+				<pre><code class="nohighlight" data-trim>
+move_field('032@.a', 'Edition')
```

Comment:
Mir kommt hier ein Punkt in den Sinn.
Wenn Felder sich wiederholen, dann kann das Ergebnis nicht so ausgewertet werden.
```suggestion
set_array("Edition")
move_field('032@.a', 'Edition.$append')
```

Reply:
Aber `032@` und `032@.a` sind ja gar nicht wiederholbar (s. https://format.k10plus.de/avram.pl?field=032@), also können wir das Beispiel hier einfach halten.